spec/ParseHooks.spec.js

```diff
@@ -464,9 +464,8 @@ describe('Hooks', () => {
           expect(err).not.toBe(undefined);
           expect(err).not.toBe(null);
           if (err) {
-            expect(err.code).toBe(Parse.Error.SCRIPT_FAILED);
-            expect(err.message.code).toEqual(1337);
```

Comment:
Looked strange before the change, but why had this to be  changed in this PR? And is that a breaking change?